### DISCUSSION TITLE
best engine

### DISCUSSION THREAD

**Discussion by suprstarrd** [FunkinCrew/Funkin#1982] [Polls]
i am here to start a war

(this is a joke i think both are good but i'm sure there are reasons to use one over the other)

**Comment by Cheemsandfriends** [comment]
What about FNF vanilla? :trollface:

**Reply by suprstarrd** [reply]
that's not an engine, that's hell

**Reply by s-zenmode** [reply]
> that's not an engine, that's hell

i mean it says it'll have improvements and robust mod support in the full game SOooOSOoSSOso

**Reply by isakube** [reply]
> > that's not an engine, that's hell
> 
> i mean it says it'll have improvements and robust mod support in the full game SOooOSOoSSOso

im in discord dms with mastereric, the final update to the demo will likely have some form of mod support. That, and **"Alexa, enable the reddit mod"**

**Comment by libauror-a** [comment]
plank engine :trollface:

**Comment by suprstarrd** [comment]
psychs at 69% rn (nice)
